content/docs/editing.md

```diff
@@ -66,11 +75,20 @@ Aside from the normal undo/redo/copy/paste entries, there are a number of sub-me
+        * NOTE: Sorting is performed with the assumption that all line-endings in the file are uniform and match the current selection for the file being edited -- the quickest way to check that selection is to glance at the status bar, where the current line-ending type is shown either as `Windows (CR LF)`, `Unix (LF)` or `Macintosh (CR)`.  It might be desirable to check the line-ending types in your file before executing a sorting operation.
+        * NOTE: If a [Column Mode](./#column-mode-column-editor) selection is active, the sort will re-order all the lines included in the selection, but the sort key (the text that decides the sort order) will be limited to what is inside the column selection. If the keys are identical on two lines, then the order of those two lines will not change (even if text outside of the selected key columns is different).
 * `Comment/Uncomment >` ⇒ submenu with actions that add or remove comment syntax, based on the file's **Language** selection
 * `Auto-Completion >` ⇒ submenu with actions that trigger auto-completion of function name, word, function parameter, and pathname, affected by [**Preferences > Auto-Completion** settings](../preferences/#auto-completion)
-* `EOL Conversion >` ⇒ submenu with actions that convert line endings between Windows (`CRLF`), Unix (`LF`), and old Macintosh (`CR`) values
-* `Blank Operations >` ⇒ submenu with actions that trim or convert spaces and TABs
+* `EOL Conversion >` ⇒ submenu with actions that convert line endings between Windows (`CRLF`), Unix (`LF`), and old Macintosh (`CR`) values; these operations affect all of the lines of the current file
+* `Blank Operations >` ⇒ submenu with actions that trim or convert spaces and tab characters on ALL lines of the current file
+    * `Trim Trailing Space`: removes any space or tab characters occurring at the end of a line, after any non-whitespace characters
+    * `Trim Leading Space`: removes any space or tab characters occurring at the beginning of a line, before any non-whitespace characters
+    * `Trim Leading and Trailing Spaces`: combines the functionalities of Trim Trailing Space` and `Trim Leading Space` into one command
+    * `EOL to Space`: replaces line-ending characters with a single space character (similar to `Join Lines` functionality, but acts upon the entire file rather than the active selection); note: "EOL" means "End Of Line" -- in other words, line-ending characters
+    * `Remove Unnecessary Blank and EOL` : performs a combined `Trim Leading and Trailing Spaces` and `EOL to Space` operation
+    * `TAB to Space` : replaces any tab characters with their equivalent number of spaces
+    * `Space to TAB (All)` : consolidates space characters into an equivalent number of tab characters, wherever the spaces occur
+    * `Space to TAB (Leading)` : consolidates space characters into an equivalent number of tab characters, but only where they occur before the first non-whitespace character on a line
```

Comment:
Somehow, for the three `TAB to Space`/`Space to TAB` items, it would be nice to indicate once that the `equivalent number of spaces (or tab characters) is based on the [Settings > Preferences > Language > Tab Settings: Tab Size](../preferences/#language) for the active language`.... but I'm not sure the best way to put that in this section while not repeating that long bit of text on all three of the lines